api/src/pcapi/routes/backoffice/accounts/blueprint.py

```diff
@@ -156,15 +159,19 @@ def anonymize_public_account(user_id: int) -> utils.BackofficeResponse:
         flash(utils.build_form_error_msg(form), "warning")
         return redirect(url_for("backoffice_web.public_accounts.get_public_account", user_id=user_id), code=303)
 
+    extract = users_api.has_unprocessed_extract(user)
+    if extract:
```

Comment:
```suggestion
    if users_api.has_unprocessed_extract(user):
```

De plus le nom de la variable était trompeur ce n'était pas un extract mais un boolean